commons/mysql/Dockerfile

```diff
@@ -2,29 +2,25 @@ FROM ubuntu:14.04
 LABEL maintainer="Mark Sutherland <mark.sutherland@epfl.ch>"
```

Comment:
This should be changed to the debian base os.

Reply:
MySQL v5.5 doesn't seem to be supported on Debian buster, check [link](http://repo.mysql.com/apt/debian/dists/buster/) 
Also, this [mysql-dump](https://github.com/parsa-epfl/cloudsuite/blob/dev/benchmarks/web-serving/db_server/files/elgg_db.dump) is for v5.5. We need to test if it gets imported successfully on later versions.

Reply:
@alok123t this PR looks fine and can be merged.
However, I think it's better if we can bump up the PHP and MySQL versions, as both are currently very old. Now that we are upgrading these benchmarks and we plan to have a new release, it makes sense to also upgrade these. This also resolves the issue with packages not being available for the base OS and prevents other issues arising in the near future. Can you do this?